packages/create-waku/tsconfig.json

```diff
@@ -1,12 +1,12 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "rootDir": "./src",
+    "rootDir": ".",
     "outDir": "./dist",
```

Comment:
Is it a convention or a standard?
I'm not super comfortable with root being `.` and out being `./dist`.